### PR TITLE
add scroll-scope v0.1.0 with autoupdate config

### LINKED ISSUE #7515: [Request] Add scroll-scope
**Library name:** scroll-scope
**Git repository url:**https://github.com/Eiskis/scroll-scope
**npm package url(optional):** 
**License(s):** unknow ?
**Official homepage:**http://eiskis.net/scroll-scope/
**Wanna say something? Leave message here:**thanks a lot

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
@x09326 please help me check this pull request, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Eiskis/scroll-scope
 * Official website (optional, not the repository): http://eiskis.net/scroll-scope/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 99
   - Count of watchers: 5
   - Count of forks: 4
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7515, cc @Eiskis

### CODE REVIEW

File: ajax/libs/scroll-scope/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "filename": "scroll-scope.min.js",
+  "description": "Small jQuery plugin to keep parent element still when scrolling an element to its boundary.",
+  "keywords": [
+    "scroll-scope",
+    "jQuery plugin",
+    "scrollable element"
+  ],
+  "repository": {
+    "type": "git",
+    "url": ""
```
Comment (x09326) [inline ajax/libs/scroll-scope/package.json]: @dannyxx001 I think you need to add the url.

**Comment by x09326** [comment]
@PeterDaveHello I think this PR is okay.
Please help me review it again, thanks.

**Comment by PeterDaveHello** [comment]
What's the reason why your content is different as upstream?

https://github.com/Eiskis/scroll-scope/blob/v0.1.0/scroll-scope.js
https://github.com/dannyxx001/cdnjs/blob/scroll-scope/ajax/libs/scroll-scope/0.1.0/scroll-scope.js

Obviously different!

**Comment by dannyxx001** [comment]
I clone from master, so the difference are master and tag 0.1.0. Should I use tag 0.1.0?

**Comment by PeterDaveHello** [comment]
@dannyxx001 If you claim it's version v0.1.0, why don't you grab files from the v0.1.0 tag but master branch? That's a basic idea we talk about when we have the hackathon, please keep it in mind.

**Comment by dannyxx001** [comment]
Sorry, I have updated it.

**Comment by PeterDaveHello** [comment]
@dannyxx001 Thanks.

**Pull request merged**